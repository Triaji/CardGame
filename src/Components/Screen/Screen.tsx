import React from 'react';
import {View, SafeAreaView} from 'react-native';

type Props = {
  children: React.ReactNode;
};

const Screen = (props: Props) => {
  const {children} = props;
  return (
    <SafeAreaView
      style={{flex: 1, flexDirection: 'column', backgroundColor: '#333'}}>
      {children}
    </SafeAreaView>
  );
};

export default Screen;
