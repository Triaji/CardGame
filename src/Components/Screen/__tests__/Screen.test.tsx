import React from 'react';
import { Text } from 'react-native';
import Screen from '../Screen';
import renderer from 'react-test-renderer';

describe('ScreenComponent', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Screen>
          <Text>Test</Text>
        </Screen>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
