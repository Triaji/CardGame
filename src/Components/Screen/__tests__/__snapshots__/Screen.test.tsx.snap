// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScreenComponent renders correctly 1`] = `
<RCTSafeAreaView
  style={
    Object {
      "backgroundColor": "#333",
      "flex": 1,
      "flexDirection": "column",
    }
  }
>
  <Text>
    Test
  </Text>
</RCTSafeAreaView>
`;
